docs/api_v2/api_v2_reference.apib

```diff
@@ -4018,53 +4034,55 @@ The user account is identified by the provided authentication token.
-    - BUYPROXIES63748:      10             (number, nullable)
-    - qfCaFFuCodXxAS59E:    3              (number, nullable)
-    - GOOGLESERP:           1              (number, nullable)
-- monthlyDatacenterProxyGbytes:      500            (number, nullable)
+- email:                 `bob@example.com`                  (string, required)
+- proxy                                                     (object, required)
+    - password:          `ad78knd9Jkjd86`                   (string, required)
+    - groups                                                (array, required)
+- plan                                                      (object, required)
+    - id:                               `Personal`          (string, required)
+    - description:                      `Cost-effective plan for freelancers, developers and students.` (string, required)
+    - isEnabled:                        true                (boolean, required)
+    - monthlyBasePriceUsd:              49                  (number, required)
+    - monthlyUsageCreditsUsd:           49                  (number, required)
+    - usageDiscountPercent:             0                   (number, required)
+    - enabledPlatformFeatures                               (array, required)
+        - `ACTORS`                                          (array, required)
+        - `STORAGE`                                         (array, required)
+        - `PROXY_SERPS`                                     (array, required)
+        - `SCHEDULER`                                       (array, required)
+        - `WEBHOOKS`                                        (array, required)
+    - maxMonthlyUsageUsd:                9999               (number, required)
+    - maxActorMemoryGbytes:              32                 (number, required)
+    - maxMonthlyActorComputeUnits:       1000               (number, required)
+    - maxMonthlyResidentialProxyGbytes:  10                 (number, required)
+    - maxMonthlyProxySerps:              30000              (number, required)
+    - maxMonthlyExternalDataTransferGbytes: 1000            (number, required)
+    - maxActorCount:                    100                 (number, required)
+    - maxActorTaskCount:                1000                (number, required)
+    - dataRetentionDays:                14                  (number, required)
+    - availableProxyGroups                                  (object, required)
+      - SOMEGROUP:                      20                  (number, required)
+      - ANOTHERGROUP:                   200                 (number, required)
+    - teamAccountSeatCount:             1                   (number, required)
+    - supportLevel:                     `COMMUNITY`         (string, required)
+    - availableAddOns                                       (array, required)
+
+## UserUsage (object)
+- usageCycle                                        (object, required)
```

Comment:
How does this work? This is a monthly usage cycle, or full billing cycle (e.g. entire year) ?

Reply:
I'm thinking, we had this philosophy of providing in API all info that we provide in UI. So user should also be able to get daily data for the platform usage. Well, we don't need to do this now, but let's say we add it in the future, so the data model should count with it... let's think this API through little more. Once we launch it, there is no going back

Reply:
I commented-out this endpoint. So if anybody will use it it's on own risk :)

Reply:
cool

Reply:
But it seems the `## Usage [/v2/users/me/usage{?token}]` section will still be in code